contracts/OctobayGovernor.sol

```diff
@@ -0,0 +1,132 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.0;
+pragma experimental ABIEncoderV2;
+
+import './OctobayStorage.sol';
+import './OctobayGovTokenFactory.sol';
+import './OctobayGovToken.sol';
+
+// This contract acts as Octobay's storage of all Governors which are used for voting on proposals
+contract OctobayGovernor is OctobayStorage {
+
+    struct Governor {
+        bool isValue; // Ensure we have a valid value in the map
+        uint256 proposalCount; // Number of proposals
+        uint16 newProposalReq; // min percentage required for a token holder to create a new proposal
+        mapping (uint => Proposal) proposalList; // List of proposals
+    }
+
+    struct Proposal {
+        bool isValue; // Ensure we have a valid value in the map
+        address creator; // Creator of this proposal
+        string discussionId; // GitHub Graph ID
+        uint256 startDate; // timestamp from when proposal is Active
+        uint256 endDate; // timestamp for when voting closes on proposal, can be 0 (open ended)
+        uint16 quorum; // min percentage (0 - 10000)
+        int16 voteCount; // the current vote count as a percent of supply
+        OctobayGovToken votingToken; // governance token required to vote
+        mapping (address => Vote) votesBySubmitter; // map of votes submitted for proposal by submitter
+    }
+
+    struct Vote {
+        bool hasVoted; // to track whether a token holder has already voted
+        int16 vote; // can be negative, represents token share (0 - 10000)
+    }
+
+    enum ProposalState {
+        Pending,
+        Active,
+        Defeated,
```

Comment:
uuhhh you mean like... having too many "downvotes" could trigger a negative quorum, so to say? yeah let's do that.